x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/settings/custom_links.spec.ts

```diff
@@ -6,10 +6,41 @@
  */
 
 const basePath = '/app/apm/settings/custom-links';
+const deleteCustomLink = () => {
+  // delete customLink if exists
+  const kibanaUrl = Cypress.env('KIBANA_URL');
+  cy.request({
+    log: false,
+    method: 'GET',
+    url: `${kibanaUrl}/internal/apm/settings/custom_links`,
+    body: {},
+    headers: {
+      'kbn-xsrf': 'e2e_test',
+    },
+    auth: { user: 'editor', pass: 'changeme' },
+  }).then((response) => {
+    response.body.customLinks.map((item: any) => {
```

Comment:
I think you'll still need to return the promise to ensure that we wait for it:

```suggestion
    return response.body.customLinks.map((item: any) => {
```

Reply:
cypress complain if I add that return, the other one is fine